### PR TITLE
Add the feature of resource-level region[2/2]

### PR DESCRIPTION
**Pull request by markyu92**
This commit contains the modification of docs about resource-level region.

### CODE REVIEW

File: docs/data-sources/antiddos.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ data "huaweicloud_antiddos" "antiddos" {
 ## Argument Reference
 The following arguments are supported:
 
+* `region` - (Optional) The region in which to obtain the Antiddos client.  If omitted, the provider-level region will work as default. Changing this creates a new resource.
```
Comment (ShiChangkuo) [inline docs/data-sources/antiddos.md]: s/Changing this creates a new resource.//

File: docs/resources/cce_addon.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ resource "huaweicloud_cce_addon" "addon_test" {
 
 ## Argument Reference
 The following arguments are supported:
+* `region` - (Optional) The region in which to obtain the cce resource. If omitted, the provider-level region will work as default. Changing this creates a new resource.
```
Comment (ShiChangkuo) [inline docs/resources/cce_addon.md]: s/the cce resource/the cce addon resource/

File: docs/data-sources/antiddos.md (hunk 1 above)
Comment (markyu92) [reply docs/data-sources/antiddos.md]: This sentence has been removed in all datasource docs which in this request.

File: docs/resources/cce_addon.md (hunk 2 above)
Comment (markyu92) [reply docs/resources/cce_addon.md]: In this request, specific resource name is added in the description in resource docs.

**Pull request merged**